srcpkgs/tcc/template

```diff
@@ -15,6 +15,13 @@ license="LGPL-2.1"
 distfiles="http://repo.or.cz/tinycc.git/snapshot/${_gitrev}.tar.gz>${pkgname}-${version}.tar.gz"
 checksum=1c3a4150b4aaab9751af2112e8bfd8a93fd1a9d36af048e996ebfc0fd1f07e48
 
+case "$XBPS_TARGET_MACHINE" in
+	*-musl) configure_args="--config-musl";;
+esac
+case "$XBPS_TARGET_MACHINE" in
+	aarch64*|armv[567]*|i686*) nostrip=yes;;
```

Comment:
no! This is only a workaround so xbps-src doesn't complain, but I highly doubt that it fixes cross builds.
`usr/bin/aarch64-linux-gnu-strip: Unable to recognise the format of the input file `/destdir/aarch64-linux-gnu/tcc-0.9.27/usr/bin/tcc'` <- this line normally indicates that the binary is not cross compiled/not in the target architecture.
The build-system would need to be patched to get it cross compiled or just set `nocross=yes`

Reply:
I'll go with nocross then